metricbeat/schema/schema.go

```diff
@@ -71,19 +71,13 @@ func (o Object) HasKey(key string) bool {
 
 // ApplyTo adds the fields extracted from data, converted using the schema, to the
 // event map.
-func (s Schema) ApplyTo(event common.MapStr, data map[string]interface{}) (common.MapStr, *Errors) {
```

Comment:
Why did you change *Errors to error? The idea is that after `Apply` the `Errors` is available and checks can be made if required or invalid fields were missing to create a more detailed error report.